eogrow/pipelines/export_maps.py

```diff
@@ -56,6 +61,13 @@ class Schema(Pipeline.Schema):
                 " With this parameter you force to always make copies."
             ),
         )
+        compress_temporally: bool = Field(
```

Comment:
This might be a bit confusing for the user. IIRC the output is a mega-tiff (a multi-band tiff containing bands and timestamps in a single tiff). From that perspective, the parameter name `compress_temporally` might be a bit confusing, especially since in the description below it says `Performs temporal split of tiffs if needed`.

I'm a bit confused as to how exactly the output should look, so I cannot give a proper suggestion, just a warning that I find the name confusing.

Reply:
I updated the description a bit.
By default you get multiple tiffs (one per timestamp) but with this enabled you get a big chungus, which has `bands * timestamps` bands (the old behaviour)